tests/DbaaS/verifyDBaaS_test.js

```diff
@@ -336,29 +336,21 @@ Scenario('PMM-T546 Verify Actions column on Kubernetes cluster page @dbaas',
 
 Scenario(
   'PMM-T969 - Verify pmm-client logs when incorrect public address is set @dbaas',
-  async ({ I, pmmSettingsPage, dbaasAPI, dbaasPage, dbaasActionsPage }) => {
-    const clusterName = 'Alyona test';
+  async ({ I, settingsAPI, dbaasAPI, dbaasPage, dbaasActionsPage }) => {
     const dbClusterName = dbaasPage.randomizeClusterName('dbcluster');
     const dbType = 'MySQL';
     const address = 'https://1.2.3.4';
     const logsText = `Registering pmm-agent on PMM Server...
 Failed to register pmm-agent on PMM Server: Post "https://https:%2F%2F1.2.3.4/v1/management/Node/Register": dial tcp: lookup ${address}: no such host.
 [u'pmm-agent', u'setup'] exited with 1.
 Restarting [u'pmm-agent', u'setup'] in 5 seconds because PMM_AGENT_SIDECAR is enabled ...`;
-
-    if (!await dbaasAPI.apiCheckRegisteredClusterExist(clusterName)) {
-      await dbaasAPI.apiRegisterCluster(process.env.kubeconfig_minikube, clusterName);
-    }
-
-    await pmmSettingsPage.openAdvancedSettings();
-    I.waitForVisible(pmmSettingsPage.fields.publicAddressInput, 30);
-    pmmSettingsPage.addPublicAddress(address);
+  
+    await settingsAPI.changeSettings({ publicAddress: address });
+    await dbaasAPI.apiCreatePXCCluster(dbClusterName, clusterName);
```

Comment:
why not use UI to do this?

Reply:
It's faster and UI way to add clusters is already covered in many previous tests so I don't think we need to repeat it with every test